### ISSUE TITLE
Unicode error with identities/migrate.py

### ISSUE THREAD

**Issue by jgbarah** [VizGrimoire/VizGrimoireUtils#4]
[I'm not sure if it is better to file this bug report here or in sortinghat, but since the affected script is here, I'm filing it here.]

When running the migration process from "old-style" affiliation/uids to sortinghat, I had this problem:

```
(grimoire-env)jgb@parato:~/src/jgbarah/Grimoire-demos$ migrate.py -u jgb -p XXX -d openstack_scm_2015_05 -s openstack_ids_2015_05 -i openstack_scm_2015_05 -o
Traceback (most recent call last):
  File "/home/jgb/src/jgbarah/Grimoire-demos/grimoire-env/bin/migrate.py", line 204, in <module>
    main()
  File "/home/jgb/src/jgbarah/Grimoire-demos/grimoire-env/bin/migrate.py", line 53, in main
    enroll_identities(db, identities, args.insert_orgs)
  File "/home/jgb/src/jgbarah/Grimoire-demos/grimoire-env/bin/migrate.py", line 110, in enroll_identities
    print msg.enconde('UTF-8')
AttributeError: 'unicode' object has no attribute 'enconde'
```

It seems there is something that the script cannot encode... Any idea?

**Comment by jgbarah** [comment]
Strange. I've reproduced the whoile migration process, but now I get some other errors, which seem reasonable:

```
migrate.py -u jgb -p XXX -d openstack_scm_2015_05 -s openstack_ids_2015_05 -i openstack_scm_2015_05 -o
Error: start date 2013-10-03 00:00:00 cannot be greater than 2013-02-01 00:00:00 - (90a93022209fee4b1a177d60fa0c8b6848759936, Cloudbase Solutions, 2013-10-03 00:00:00, 2013-02-01 00:00:00)
Error: start date 2013-10-03 00:00:00 cannot be greater than 2013-02-01 00:00:00 - (90a93022209fee4b1a177d60fa0c8b6848759936, Cloudbase Solutions, 2013-10-03 00:00:00, 2013-02-01 00:00:00)
Error: start date 2014-08-28 20:53:52 cannot be greater than 2014-06-18 00:00:00 - (bf0edb054ec85d4353f47185536d3c4d598b2e9b, eNovance, 2014-08-28 20:53:52, 2014-06-18 00:00:00)
Error: start date 2014-08-28 20:53:52 cannot be greater than 2014-06-18 00:00:00 - (bf0edb054ec85d4353f47185536d3c4d598b2e9b, eNovance, 2014-08-28 20:53:52, 2014-06-18 00:00:00)
Error: start date 2014-07-23 10:24:50 cannot be greater than 2014-06-18 00:00:00 - (482ba0f93675ebc86ef5eced6df1851c1185d233, eNovance, 2014-07-23 10:24:50, 2014-06-18 00:00:00)
Error: start date 2014-07-23 10:24:50 cannot be greater than 2014-06-18 00:00:00 - (482ba0f93675ebc86ef5eced6df1851c1185d233, eNovance, 2014-07-23 10:24:50, 2014-06-18 00:00:00)
```

No idea why why it didn't fail with the unicode error now... (?)

**Comment by sduenas** [comment]
You were using an older version of the script that had a typo. In fact, you fixed it on #3 :). I close this ticket.

**Comment by jgbarah** [comment]
Awesome! Thxs a lot. I guess I need some more storage space in my brain... :-(
